src/app/models/blocks/index.js

```diff
@@ -3,7 +3,7 @@ export const imageType = 'image';
 export const videoType = 'video';
 export const rawImageType = 'rawImage';
 
-export const singleTextBlock = text => ({
+export const singleTextBlock = (text, id = null) => ({
```

Comment:
a bit concerned about the `id's` appearing as null in snaps, perhaps we could set it to `"testId"` but looks good generally

Reply:
Now using `testId` as id in src/app/models/blocks/index.test.js